requirements-dev.txt

```diff
@@ -21,3 +21,4 @@ isort
 pipdeptree
 eth-typing<2.0.0
 eth-abi<=1.2.2
+six>=1.12
```

Comment:
What's that for?

Reply:
CI fails without it.  Somehow `master` is also broken.  Maybe merging this alone https://github.com/raiden-network/raiden-contracts/pull/1264 is the better way ahead.

Reply:
#1261 applied the same change.